### PR TITLE
Bump up dependencies to fix cves

### PR DESCRIPTION
**Pull request by kezhenxu94**
Major changes are upgrade mybatis-plus (I hate mybatis-plus) and adjust the codes accordingly, others are spotless format

### CODE REVIEW

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/datasource/SpringConnectionFactory.java [hunk 1]
```diff
@@ -51,8 +53,10 @@ public class SpringConnectionFactory {
     public DataSourceScriptDatabaseInitializer dataSourceScriptDatabaseInitializer;
 
     @Bean
-    public PaginationInterceptor paginationInterceptor() {
-        return new PaginationInterceptor();
+    public MybatisPlusInterceptor paginationInterceptor() {
+        MybatisPlusInterceptor interceptor = new MybatisPlusInterceptor();
+        interceptor.addInnerInterceptor(new PaginationInnerInterceptor(DbType.MYSQL));
```
Comment (caishunfeng) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/datasource/SpringConnectionFactory.java]: Don't other databases need to be handled?

Comment (kezhenxu94) [reply dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/datasource/SpringConnectionFactory.java]: > Don't other databases need to be handled?

Uh... forgot to pass the db type here, now fixed

**Comment by kezhenxu94** [comment]
@caishunfeng @ruanwenjun @SbloodyS @EricGao888 this PR is ready now

File: docs/docs/zh/faq.md [hunk 2]
```diff
@@ -504,33 +527,49 @@ A： 1，用户修改了 api server 配置文件中的![apiServerContextPath](ht
...
 A：1，编辑 ngnix 配置文件 vi /etc/nginx/nginx.conf，更改上传大小 client_max_body_size 1024m。
 
-   ​	2，更新 google chrome 版本到最新版本。
+<<<<<<< HEAD
+​	2，更新 google chrome 版本到最新版本。
+=============================
+
+​	2，更新 google chrome 版本到最新版本。
+
+>>>>>>> 85200811c (Bump up dependencies to fix cves)
```
Comment (ruanwenjun) [inline docs/docs/zh/faq.md]:
```suggestion
​	2，更新 google chrome 版本到最新版本。
```

File: docs/docs/zh/faq.md [hunk 3]
```diff
@@ -504,33 +527,49 @@ A： 1，用户修改了 api server 配置文件中的![apiServerContextPath](ht
...
 
 ## Q：DS 后端接口文档
+
+<<<<<<< HEAD
 A：1，http://106.75.43.194:8888/dolphinscheduler/swagger-ui/index.html?language=zh_CN&lang=zh。
+============================================================================================
 
+>>>>>>> 85200811c (Bump up dependencies to fix cves)
+
+A：1，http://106.75.43.194:8888/dolphinscheduler/doc.html?language=zh_CN&lang=zh。
```
Comment (ruanwenjun) [inline docs/docs/zh/faq.md]:
```suggestion
A：1，http://106.75.43.194:8888/dolphinscheduler/swagger-ui/index.html?language=zh_CN&lang=zh。
```

File: docs/docs/zh/faq.md [hunk 4]
```diff
@@ -582,13 +621,26 @@ sed -i 's/Defaults    requirett/#Defaults    requirett/g' /etc/sudoers
 ---
 
 ## Q：Yarn多集群支持
+
 A：将Worker节点分别部署至多个Yarn集群，步骤如下（例如AWS EMR）：
 
-   1. 将 Worker 节点部署至 EMR 集群的 Master 节点
+<<<<<<< HEAD
```
Comment (ruanwenjun) [inline docs/docs/zh/faq.md]: Please resolve the conflicts.

**Review by ruanwenjun — CHANGES_REQUESTED** [review]

**Comment by caishunfeng** [comment]
> SonarCloud Quality Gate failed.    [Quality Gate failed](https://sonarcloud.io/dashboard?id=apache-dolphinscheduler&pullRequest=11765)
> 
> [Bug](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=BUG) [C](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=BUG) [4 Bugs](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=CODE_SMELL) [11 Code Smells](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=11765&resolved=false&types=CODE_SMELL)
> 
> [](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=11765&metric=new_coverage&view=list) [50.7% Coverage](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=11765&metric=new_coverage&view=list) [](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=11765&metric=new_duplicated_lines_density&view=list) [0.7% Duplication](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=11765&metric=new_duplicated_lines_density&view=list)

Please check the sonar result.

**Review by ruanwenjun — DISMISSED** [review]
LGTM, please solve the conflicts.

**Review by ruanwenjun — APPROVED** [review]

**Pull request merged**